SA3/qa_web/templates/qa_web/answerspage.html

```diff
@@ -107,7 +108,7 @@ <h3>Other answers:</h3>
     {% for answer in answers %}
         <form id="select{{ answer.id }}_form" method="post">
             {% csrf_token %}
-            <p>{{ answer.content }}</p>
+            <p><span id="marked4">{{ answer.content }}</span></p>
```

Comment:
This will result in multiple tags with the same id, maybe we could use the answer.id value to generate a unique id for each span and dynamically access them in the javascript? I am not sure if this specific id serves a particuliar purpose outside of the prepare_markdown function of base.js and therefore might be wrong

Reply:
You are right, I completely forgot that some spans were within for loops. The easier fix would be to specify the `class` as marked everywhere and not touch `id` for spans.

Reply:
If that works then I agree it is the best solution